includes/class-wc-logger.php

```diff
@@ -64,16 +64,16 @@ public function should_handle( $level ) {
 	/**
 	 * Add a log entry.
 	 *
-	 * @deprecated 2.7.0
+	 * This is not the preferred method for adding log messages. Please use log() or any one of
+	 * the level methods (debug(), info(), etc.). This method may be deprecated in the future.
 	 *
 	 * @param string $handle
 	 * @param string $message
 	 * @return bool
 	 */
-	public function add( $handle, $message ) {
-		wc_deprecated_function( 'WC_Logger::add', '2.7', 'WC_Logger::log' );
+	public function add( $handle, $message, $level=WC_Log_Levels::NOTICE ) {
```

Comment:
I think that's from an outdated commit, the current branch looks correct: https://github.com/woocommerce/woocommerce/blob/wc-logger/includes/class-wc-logger.php#L81